starter_code/views/index.hbs

```diff
@@ -1,3 +1,10 @@
+
+
 <div class="index-container">
-  
+  <img src="/images/beer.png" class="rounded mx-auto d-block">
+  <br>
+<a class="btn btn-sm press" type="submit" href="/beers" >Check the beers</a>
+<br>
+<br>
+<a class="btn btn-sm press" type="submit" href="/beers" >Check a random beer</a>
```

Comment:
your random beers button is linking to your beer list. The href should be to randomBeer.  ;)